helm-configs/nova/nova-helm-overrides.yaml

```diff
@@ -863,7 +863,16 @@ conf:
     filter:audit:
       paste.filter_factory: keystonemiddleware.audit:filter_factory
       audit_map_file: /etc/nova/api_audit_map.conf
-  policy: {}
+  policy:
+    "os_compute_api:os-lock-server:unlock:unlock_override": "!"
+    "os_compute_api:os-deferred-delete:force": "rule:context_is_admin"
+    "os_compute_api:servers:migrations:force_complete": "!"
+    "os_compute_api:os-pause-server:pause": "rule:context_is_admin"
+    "os_compute_api:os-pause-server:unpause": "rule:context_is_admin"
+    "os_compute_api:os-suspend-server:resume": "rule:context_is_admin"
+    "os_compute_api:os-suspend-server:suspend": "rule:context_is_admin"
+    "os_compute_api:os-shelve:shelve": "rule:context_is_admin"
```

Comment:
I think we should allow the user to shelve and unshelve `admin_or_owner`.

Reply:
This was bases on the conversation in teams channel that admins can use shelve to force suspend delinquent account since suspend action makes it hard to migrate etc. If this is not the case then I agree.

One thing to note might be that Enterprise customers might want this, so I am not sure exactly how to handle that perhaps move this to prod-override file or something similar ?

Reply:
I think this is good as-is.